lib/objects/objectsInMemServerRedis.js

```diff
@@ -590,6 +591,28 @@ class ObjectsInMemoryServer extends ObjectsInMemoryFileDB {
             }
         });
 
+        handler.on('exists', async (data, responseId) => {
+            if (!data || !data.length) {
+                handler.sendInteger(responseId, 0);
+            }
+
+            const {id, namespace, name} = this._normalizeId(data[0]);
+
+            if (namespace === this.namespaceObj) {
+                try {
+                    const obj = await this.getObjectAsync(id);
+                    handler.sendInteger(responseId, obj ? 1 : 0);
+                } catch (e) {
+                    return handler.sendError(responseId, e);
+                }
+            } else if (namespace === this.namespaceFile) {
+                // @ts-ignore
+                handler.sendInteger(responseId, +await this.fileExists(id, name));
```

Comment:
typo? there is a "+"

Reply:
No, file exists returns a boolean, but we want to simulate redis, thus we send 0 or 1 as an integer. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Unary_plus

Reply:
ok, I did not know that operator :-)